Forge/src/main/java/at/petrak/hexcasting/forge/ForgeHexConfig.java

```diff
@@ -175,6 +176,10 @@ public Server(ForgeConfigSpec.Builder builder) {
                     "Resource locations of disallowed actions. Trying to cast one of these will result in a mishap.")
                 .defineList("actionDenyList", List.of(), Server::isValidReslocArg);
 
+            greaterTeleportSplatsItems = builder.comment(
+                    "Should greater teleport splat players inventory?"
```

Comment:
```suggestion
                    "Should items fly out of the player's inventory when using Greater Teleport?"
```

imo since there's no title here, it's better to say exactly what it does (copied from Fabric) than to keep the whimsical "splat"